### ISSUE TITLE
Problem with rank Order.

### ISSUE THREAD

**Issue by davidvilanova** [bioinformatics-centre/kaiju#41]
Hi ,
while running kaiju i noticed a problem with Class and order taxonomy. See the following example

```
C       k141_1685_2     169292  689     169292, WP_049377218.1, TELRRFRSDQGVKPSQKVPGRLDFAAADLAGQEELVRNLANTTAPGEDFDPSASIEVRLSQATVEVTLDTHGAVDVEAERKRLEKDLAKANKELEQTGKKLGNENFLSKAPEEVVNK
IKERQQIAREEVERITSRLEGLK,        Actinobacteria; Corynebacteriales; Actinobacteria; Corynebacteriaceae; Corynebacterium; Corynebacterium aurimucosum; 
```
NCBi taxid 169292 corresponds to :
`cellular organisms; Bacteria; Terrabacteria group; Actinobacteria; Actinobacteria; Corynebacteriales; Corynebacteriaceae; Corynebacterium;Corynebacterium aurimucosum`

I´m trying to get "Phylum","Class","Order","Family","Genus","Species" through the command
`addTaxonNames -t nodes.dmp -n names.dmp -i kaiju_output.ncbi_id -r phylum,order,class,family,genus,species 
`

The rank **order** based on NCBI is `Corynebacteriales` but Kaiju returns `Actinobacteria`

Maybe i´m doing something wrong ?? I have downloaded taxdump.tar.gz as suggested.

**Comment by pmenzel** [comment]
Yeah, check the command you are running: order and class are switched.
I messed that up in the README, but it was fixed in this commit 508406578120a2beef81e70b7e7f0e4ab23d14d0 some time ago.

**Comment by davidvilanova** [comment]
great , thanks for the update
